src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -559,6 +559,7 @@ public abstract partial class ReferenceHandler
     {
         protected ReferenceHandler() { }
         public static System.Text.Json.Serialization.ReferenceHandler Preserve { get { throw null; } }
+        public static System.Text.Json.Serialization.ReferenceHandler IgnoreCycle { get { throw null; } }
```

Comment:
Why not IgnoreCycle**s**?

Reply:
Because `Preserve` is already a singular verb since it talks about what's the behavior to take when we step into a reference type. That's why I thought `IgnoreCycle` was more appropriate.

Reply:
I think I would also prefer `IgnoreCycles` here.

Reply:
+1 on the plural `IgnoreCycles`

For example, one could read `Preserve` as `PreserveDuplicates`, which would be plural too.

Reply:
Do we need to go through API review to approve the name change?
cc @terrajobst @bartonjs

Reply:
> Do we need to go through API review to approve the name change?

Yes, please.

Reply:
That's an expensive addition of an `s` 😜